ir/enum/dispflag_string.go

```diff
@@ -37,7 +37,7 @@ var (
 
 func (i DISPFlag) String() string {
 	switch {
-	case 0 <= i && i <= 2:
+	case i <= 2:
```

Comment:
These files are automatically generated, so the fix would have to go into `x/tools/cmd/stringer`. Otherwise, next time we run `go generate` the same issue would re-appear. Feel free to submit a PR or create an issue for x/tools/cmd/stringer. They use the same issue tracker as the main Go project.